Reproducible Failure in org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests#testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped

Found this while stability testing frozen tier work. In Current master:

```
./gradlew ':x-pack:plugin:searchable-snapshots:internalClusterTest' --tests "org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests.testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped" -Dtests.seed=5800A43D869703D3 -Dtests.locale=ms-MY -Dtests.timezone=America/Cuiaba -Druntime.java=14
```

fails with:

```
[2021-02-05T11:31:36,422][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] before test
[2021-02-05T11:31:36,423][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] [SearchableSnapshotsCanMatchOnCoordinatorIntegTests#testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped]: setting up test
[2021-02-05T11:31:36,423][INFO ][o.e.t.InternalTestCluster] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] Setup InternalTestCluster [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster] with seed [7F7EAA484721C64D] using [0] dedicated masters, [0] (data) nodes and [0] coord only nodes (master nodes are [auto-managed])
[2021-02-05T11:31:36,423][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] [SearchableSnapshotsCanMatchOnCoordinatorIntegTests#testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped]: all set up test
[2021-02-05T11:31:36,424][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] version[8.0.0], pid[170115], build[unknown/unknown/unknown/unknown], OS[Linux/5.8.0-41-generic/amd64], JVM[Oracle Corporation/OpenJDK 64-Bit Server VM/14/14+36-1461]
[2021-02-05T11:31:36,424][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] JVM home [/usr/lib/jvm/jdk-14]
[2021-02-05T11:31:36,424][DEPRECATION][o.e.d.n.Node             ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] data_stream.dataset="elasticsearch.deprecation" data_stream.namespace="default" data_stream.type="logs" ecs.version="1.7" elasticsearch.event.category="other" event.code="no-jdk" message="no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release"
[2021-02-05T11:31:36,424][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] JVM arguments [-agentlib:jdwp=transport=dt_socket,address=127.0.0.1:46899,suspend=y,server=n, -ea, -Djava.locale.providers=SPI,COMPAT, -Dtests.seed=5800A43D869703D3, -Didea.test.cyclic.buffer.size=1048576, -javaagent:/home/brownbear/idea/plugins/Groovy/lib/agent/gragent.jar, -javaagent:/home/brownbear/idea/plugins/java/lib/rt/debugger-agent.jar, -agentpath:/tmp/libmemory_agent.so=, -Dfile.encoding=UTF-8]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] no modules loaded
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.index.mapper.MockFieldFilterPlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.node.NodeMocksPlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.script.MockScriptService$TestPlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.snapshots.mockstore.MockRepository$Plugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.ESIntegTestCase$AssertActionNamePlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.ESIntegTestCase$TestSeedPlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.MockHttpTransport$TestPlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.TestGeoShapeFieldMapperPlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.transport.MockTransportService$TestPlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.transport.nio.MockNioTransportPlugin]
[2021-02-05T11:31:36,424][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.xpack.searchablesnapshots.LocalStateSearchableSnapshots]
[2021-02-05T11:31:36,425][INFO ][o.e.e.NodeEnvironment    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] using [2] data paths, mounts [[/ (/dev/nvme0n1p2)]], net usable_space [124.8gb], net total_space [467.9gb], types [ext4]
[2021-02-05T11:31:36,425][INFO ][o.e.e.NodeEnvironment    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] heap size [15.6gb], compressed ordinary object pointers [true]
[2021-02-05T11:31:36,426][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] node name [node_t0], node ID [YY0YoUJzQvap0Pfc3efM1A], cluster name [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster], roles [master]
[2021-02-05T11:31:36,437][INFO ][o.e.d.DiscoveryModule    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] using discovery type [zen] and seed hosts providers [settings, file]
[2021-02-05T11:31:36,447][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] initialized
[2021-02-05T11:31:36,447][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#1]]] starting ...
[2021-02-05T11:31:36,449][INFO ][o.e.t.TransportService   ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#1]]] publish_address {127.0.0.1:37113}, bound_addresses {[::1]:34385}, {127.0.0.1:37113}
[2021-02-05T11:31:36,456][INFO ][o.e.c.c.Coordinator      ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#1]]] setting initial configuration to VotingConfiguration{YY0YoUJzQvap0Pfc3efM1A}
[2021-02-05T11:31:36,501][INFO ][o.e.c.s.MasterService    ] [node_t0] elected-as-master ([1] nodes joined)[{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true} elect leader, _BECOME_MASTER_TASK_, _FINISH_ELECTION_], term: 1, version: 1, delta: master node changed {previous [], current [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}]}
[2021-02-05T11:31:36,504][INFO ][o.e.c.c.CoordinationState] [node_t0] cluster UUID set to [yhxd0PjaQ9-hZVv_1y4IUg]
[2021-02-05T11:31:36,508][INFO ][o.e.c.s.ClusterApplierService] [node_t0] master node changed {previous [], current [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}]}, term: 1, version: 1, reason: Publication{term=1, version=1}
[2021-02-05T11:31:36,509][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#1]]] started
[2021-02-05T11:31:36,510][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] version[8.0.0], pid[170115], build[unknown/unknown/unknown/unknown], OS[Linux/5.8.0-41-generic/amd64], JVM[Oracle Corporation/OpenJDK 64-Bit Server VM/14/14+36-1461]
[2021-02-05T11:31:36,510][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] JVM home [/usr/lib/jvm/jdk-14]
[2021-02-05T11:31:36,510][DEPRECATION][o.e.d.n.Node             ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] data_stream.dataset="elasticsearch.deprecation" data_stream.namespace="default" data_stream.type="logs" ecs.version="1.7" elasticsearch.event.category="other" event.code="no-jdk" message="no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release"
[2021-02-05T11:31:36,510][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] JVM arguments [-agentlib:jdwp=transport=dt_socket,address=127.0.0.1:46899,suspend=y,server=n, -ea, -Djava.locale.providers=SPI,COMPAT, -Dtests.seed=5800A43D869703D3, -Didea.test.cyclic.buffer.size=1048576, -javaagent:/home/brownbear/idea/plugins/Groovy/lib/agent/gragent.jar, -javaagent:/home/brownbear/idea/plugins/java/lib/rt/debugger-agent.jar, -agentpath:/tmp/libmemory_agent.so=, -Dfile.encoding=UTF-8]
[2021-02-05T11:31:36,510][INFO ][o.e.g.GatewayService     ] [node_t0] recovered [0] indices into cluster_state
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] no modules loaded
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.index.mapper.MockFieldFilterPlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.node.NodeMocksPlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.script.MockScriptService$TestPlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.snapshots.mockstore.MockRepository$Plugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.ESIntegTestCase$AssertActionNamePlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.ESIntegTestCase$TestSeedPlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.MockHttpTransport$TestPlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.TestGeoShapeFieldMapperPlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.transport.MockTransportService$TestPlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.transport.nio.MockNioTransportPlugin]
[2021-02-05T11:31:36,510][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.xpack.searchablesnapshots.LocalStateSearchableSnapshots]
[2021-02-05T11:31:36,512][INFO ][o.e.e.NodeEnvironment    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] using [2] data paths, mounts [[/ (/dev/nvme0n1p2)]], net usable_space [124.8gb], net total_space [467.9gb], types [ext4]
[2021-02-05T11:31:36,512][INFO ][o.e.e.NodeEnvironment    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] heap size [15.6gb], compressed ordinary object pointers [true]
[2021-02-05T11:31:36,513][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] node name [node_t1], node ID [q7qx8mNZRmSRUImgdR__OA], cluster name [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster], roles []
[2021-02-05T11:31:36,525][INFO ][o.e.d.DiscoveryModule    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] using discovery type [zen] and seed hosts providers [settings, file]
[2021-02-05T11:31:36,535][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] initialized
[2021-02-05T11:31:36,535][INFO ][o.e.l.LicenseService     ] [node_t0] license [381b041d-765b-487b-81da-6801db872f54] mode [trial] - valid
[2021-02-05T11:31:36,536][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#2]]] starting ...
[2021-02-05T11:31:36,538][INFO ][o.e.t.TransportService   ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#2]]] publish_address {127.0.0.1:37261}, bound_addresses {[::1]:34207}, {127.0.0.1:37261}
[2021-02-05T11:31:36,548][INFO ][o.e.c.s.MasterService    ] [node_t0] node-join[{node_t1}{q7qx8mNZRmSRUImgdR__OA}{i3GoskLyTASEGkfc5SMaMA}{127.0.0.1}{127.0.0.1:37261}{xpack.installed=true} join existing leader], term: 1, version: 4, delta: added {{node_t1}{q7qx8mNZRmSRUImgdR__OA}{i3GoskLyTASEGkfc5SMaMA}{127.0.0.1}{127.0.0.1:37261}{xpack.installed=true}}
[2021-02-05T11:31:36,549][INFO ][o.e.c.s.ClusterApplierService] [node_t1] master node changed {previous [], current [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}]}, added {{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}, term: 1, version: 4, reason: ApplyCommitRequest{term=1, version=4, sourceNode={node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}
[2021-02-05T11:31:36,560][INFO ][o.e.l.LicenseService     ] [node_t1] license [381b041d-765b-487b-81da-6801db872f54] mode [trial] - valid
[2021-02-05T11:31:36,560][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#2]]] started
[2021-02-05T11:31:36,560][INFO ][o.e.c.s.ClusterApplierService] [node_t0] added {{node_t1}{q7qx8mNZRmSRUImgdR__OA}{i3GoskLyTASEGkfc5SMaMA}{127.0.0.1}{127.0.0.1:37261}{xpack.installed=true}}, term: 1, version: 4, reason: Publication{term=1, version=4}
[2021-02-05T11:31:36,562][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] version[8.0.0], pid[170115], build[unknown/unknown/unknown/unknown], OS[Linux/5.8.0-41-generic/amd64], JVM[Oracle Corporation/OpenJDK 64-Bit Server VM/14/14+36-1461]
[2021-02-05T11:31:36,562][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] JVM home [/usr/lib/jvm/jdk-14]
[2021-02-05T11:31:36,562][DEPRECATION][o.e.d.n.Node             ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] data_stream.dataset="elasticsearch.deprecation" data_stream.namespace="default" data_stream.type="logs" ecs.version="1.7" elasticsearch.event.category="other" event.code="no-jdk" message="no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release"
[2021-02-05T11:31:36,562][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] JVM arguments [-agentlib:jdwp=transport=dt_socket,address=127.0.0.1:46899,suspend=y,server=n, -ea, -Djava.locale.providers=SPI,COMPAT, -Dtests.seed=5800A43D869703D3, -Didea.test.cyclic.buffer.size=1048576, -javaagent:/home/brownbear/idea/plugins/Groovy/lib/agent/gragent.jar, -javaagent:/home/brownbear/idea/plugins/java/lib/rt/debugger-agent.jar, -agentpath:/tmp/libmemory_agent.so=, -Dfile.encoding=UTF-8]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] no modules loaded
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.index.mapper.MockFieldFilterPlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.node.NodeMocksPlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.script.MockScriptService$TestPlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.snapshots.mockstore.MockRepository$Plugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.ESIntegTestCase$AssertActionNamePlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.ESIntegTestCase$TestSeedPlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.MockHttpTransport$TestPlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.TestGeoShapeFieldMapperPlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.transport.MockTransportService$TestPlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.transport.nio.MockNioTransportPlugin]
[2021-02-05T11:31:36,562][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.xpack.searchablesnapshots.LocalStateSearchableSnapshots]
[2021-02-05T11:31:36,564][INFO ][o.e.e.NodeEnvironment    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] using [2] data paths, mounts [[/ (/dev/nvme0n1p2)]], net usable_space [124.8gb], net total_space [467.9gb], types [ext4]
[2021-02-05T11:31:36,564][INFO ][o.e.e.NodeEnvironment    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] heap size [15.6gb], compressed ordinary object pointers [true]
[2021-02-05T11:31:36,565][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] node name [node_t2], node ID [H6dFSh5mRGy-Kn9DcgnAjg], cluster name [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster], roles [data]
[2021-02-05T11:31:36,576][INFO ][o.e.d.DiscoveryModule    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] using discovery type [zen] and seed hosts providers [settings, file]
[2021-02-05T11:31:36,585][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] initialized
[2021-02-05T11:31:36,586][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#3]]] starting ...
[2021-02-05T11:31:36,586][INFO ][o.e.x.s.c.PersistentCache] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#3]]] persistent cache index loaded
[2021-02-05T11:31:36,588][INFO ][o.e.t.TransportService   ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#3]]] publish_address {127.0.0.1:40527}, bound_addresses {[::1]:38239}, {127.0.0.1:40527}
[2021-02-05T11:31:36,598][INFO ][o.e.c.s.MasterService    ] [node_t0] node-join[{node_t2}{H6dFSh5mRGy-Kn9DcgnAjg}{IVAzxpolRRyuFAiB2iwK5g}{127.0.0.1}{127.0.0.1:40527}{d}{xpack.installed=true} join existing leader], term: 1, version: 5, delta: added {{node_t2}{H6dFSh5mRGy-Kn9DcgnAjg}{IVAzxpolRRyuFAiB2iwK5g}{127.0.0.1}{127.0.0.1:40527}{d}{xpack.installed=true}}
[2021-02-05T11:31:36,599][INFO ][o.e.c.s.ClusterApplierService] [node_t1] added {{node_t2}{H6dFSh5mRGy-Kn9DcgnAjg}{IVAzxpolRRyuFAiB2iwK5g}{127.0.0.1}{127.0.0.1:40527}{d}{xpack.installed=true}}, term: 1, version: 5, reason: ApplyCommitRequest{term=1, version=5, sourceNode={node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}
[2021-02-05T11:31:36,599][INFO ][o.e.c.s.ClusterApplierService] [node_t2] master node changed {previous [], current [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}]}, added {{node_t1}{q7qx8mNZRmSRUImgdR__OA}{i3GoskLyTASEGkfc5SMaMA}{127.0.0.1}{127.0.0.1:37261}{xpack.installed=true},{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}, term: 1, version: 5, reason: ApplyCommitRequest{term=1, version=5, sourceNode={node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}
[2021-02-05T11:31:36,611][INFO ][o.e.l.LicenseService     ] [node_t2] license [381b041d-765b-487b-81da-6801db872f54] mode [trial] - valid
[2021-02-05T11:31:36,611][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#3]]] started
[2021-02-05T11:31:36,611][INFO ][o.e.c.s.ClusterApplierService] [node_t0] added {{node_t2}{H6dFSh5mRGy-Kn9DcgnAjg}{IVAzxpolRRyuFAiB2iwK5g}{127.0.0.1}{127.0.0.1:40527}{d}{xpack.installed=true}}, term: 1, version: 5, reason: Publication{term=1, version=5}
[2021-02-05T11:31:36,613][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] version[8.0.0], pid[170115], build[unknown/unknown/unknown/unknown], OS[Linux/5.8.0-41-generic/amd64], JVM[Oracle Corporation/OpenJDK 64-Bit Server VM/14/14+36-1461]
[2021-02-05T11:31:36,613][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] JVM home [/usr/lib/jvm/jdk-14]
[2021-02-05T11:31:36,613][DEPRECATION][o.e.d.n.Node             ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] data_stream.dataset="elasticsearch.deprecation" data_stream.namespace="default" data_stream.type="logs" ecs.version="1.7" elasticsearch.event.category="other" event.code="no-jdk" message="no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release"
[2021-02-05T11:31:36,613][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] JVM arguments [-agentlib:jdwp=transport=dt_socket,address=127.0.0.1:46899,suspend=y,server=n, -ea, -Djava.locale.providers=SPI,COMPAT, -Dtests.seed=5800A43D869703D3, -Didea.test.cyclic.buffer.size=1048576, -javaagent:/home/brownbear/idea/plugins/Groovy/lib/agent/gragent.jar, -javaagent:/home/brownbear/idea/plugins/java/lib/rt/debugger-agent.jar, -agentpath:/tmp/libmemory_agent.so=, -Dfile.encoding=UTF-8]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] no modules loaded
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.index.mapper.MockFieldFilterPlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.node.NodeMocksPlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.script.MockScriptService$TestPlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.snapshots.mockstore.MockRepository$Plugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.ESIntegTestCase$AssertActionNamePlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.ESIntegTestCase$TestSeedPlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.MockHttpTransport$TestPlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.TestGeoShapeFieldMapperPlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.test.transport.MockTransportService$TestPlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.transport.nio.MockNioTransportPlugin]
[2021-02-05T11:31:36,613][INFO ][o.e.p.PluginsService     ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] loaded plugin [org.elasticsearch.xpack.searchablesnapshots.LocalStateSearchableSnapshots]
[2021-02-05T11:31:36,615][INFO ][o.e.e.NodeEnvironment    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] using [2] data paths, mounts [[/ (/dev/nvme0n1p2)]], net usable_space [124.8gb], net total_space [467.9gb], types [ext4]
[2021-02-05T11:31:36,615][INFO ][o.e.e.NodeEnvironment    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] heap size [15.6gb], compressed ordinary object pointers [true]
[2021-02-05T11:31:36,616][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] node name [node_t3], node ID [9T-YE4bRRZaiStnhF3Nupg], cluster name [TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster], roles [data]
[2021-02-05T11:31:36,626][INFO ][o.e.x.s.c.SharedBytes    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] creating shared snapshot cache file [size=4194304, path=/tmp/org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests_5800A43D869703D3-003/tempDir-002/node_t3/d0/snap_cache]
[2021-02-05T11:31:36,629][INFO ][o.e.d.DiscoveryModule    ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] using discovery type [zen] and seed hosts providers [settings, file]
[2021-02-05T11:31:36,639][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] initialized
[2021-02-05T11:31:36,639][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#4]]] starting ...
[2021-02-05T11:31:36,640][INFO ][o.e.x.s.c.PersistentCache] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#4]]] persistent cache index loaded
[2021-02-05T11:31:36,641][INFO ][o.e.t.TransportService   ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#4]]] publish_address {127.0.0.1:33617}, bound_addresses {[::1]:45339}, {127.0.0.1:33617}
[2021-02-05T11:31:36,651][INFO ][o.e.c.s.MasterService    ] [node_t0] node-join[{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true} join existing leader], term: 1, version: 6, delta: added {{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true}}
[2021-02-05T11:31:36,652][INFO ][o.e.c.s.ClusterApplierService] [node_t1] added {{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true}}, term: 1, version: 6, reason: ApplyCommitRequest{term=1, version=6, sourceNode={node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}
[2021-02-05T11:31:36,652][INFO ][o.e.c.s.ClusterApplierService] [node_t2] added {{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true}}, term: 1, version: 6, reason: ApplyCommitRequest{term=1, version=6, sourceNode={node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}
[2021-02-05T11:31:36,652][INFO ][o.e.c.s.ClusterApplierService] [node_t3] master node changed {previous [], current [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}]}, added {{node_t1}{q7qx8mNZRmSRUImgdR__OA}{i3GoskLyTASEGkfc5SMaMA}{127.0.0.1}{127.0.0.1:37261}{xpack.installed=true},{node_t2}{H6dFSh5mRGy-Kn9DcgnAjg}{IVAzxpolRRyuFAiB2iwK5g}{127.0.0.1}{127.0.0.1:40527}{d}{xpack.installed=true},{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}, term: 1, version: 6, reason: ApplyCommitRequest{term=1, version=6, sourceNode={node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}
[2021-02-05T11:31:36,664][INFO ][o.e.l.LicenseService     ] [node_t3] license [381b041d-765b-487b-81da-6801db872f54] mode [trial] - valid
[2021-02-05T11:31:36,664][INFO ][o.e.n.Node               ] [[test_TEST-TEST_WORKER_VM=[--not-gradle--]-CLUSTER_SEED=[9186967517290153549]-HASH=[96DDCB9CD9E]-cluster[T#4]]] started
[2021-02-05T11:31:36,664][INFO ][o.e.c.s.ClusterApplierService] [node_t0] added {{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true}}, term: 1, version: 6, reason: Publication{term=1, version=6}
[2021-02-05T11:31:36,668][INFO ][o.e.c.m.MetadataCreateIndexService] [node_t0] [zddkaqnczh] creating index, cause [api], templates [], shards [3]/[0]
[2021-02-05T11:31:36,694][INFO ][o.e.c.r.a.AllocationService] [node_t0] current.health="GREEN" message="Cluster health status changed from [YELLOW] to [GREEN] (reason: [shards started [[zddkaqnczh][2], [zddkaqnczh][0]]])." previous.health="YELLOW" reason="shards started [[zddkaqnczh][2], [zddkaqnczh][0]]"
[2021-02-05T11:31:36,706][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] Index [917] docs async: [false] bulk: [true] partitions [3]
[2021-02-05T11:31:36,742][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] --> creating repository [xtlqnetcyh] [mock]
[2021-02-05T11:31:36,742][INFO ][o.e.s.m.MockRepository   ] [node_t0] starting mock repository with random prefix default
[2021-02-05T11:31:36,748][INFO ][o.e.s.m.MockRepository   ] [node_t3] starting mock repository with random prefix default
[2021-02-05T11:31:36,748][INFO ][o.e.s.m.MockRepository   ] [node_t2] starting mock repository with random prefix default
[2021-02-05T11:31:36,748][INFO ][o.e.s.m.MockRepository   ] [node_t0] starting mock repository with random prefix default
[2021-02-05T11:31:36,749][INFO ][o.e.r.RepositoriesService] [node_t0] put repository [xtlqnetcyh]
[2021-02-05T11:31:36,756][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] --> creating snapshot [snapshot-1] of [zddkaqnczh] in [xtlqnetcyh]
[2021-02-05T11:31:36,758][INFO ][o.e.s.SnapshotsService   ] [node_t0] snapshot [xtlqnetcyh:snapshot-1/MLdeB_rfTHSm1Pi94e247A] started
[2021-02-05T11:31:36,776][INFO ][o.e.s.SnapshotsService   ] [node_t0] snapshot [xtlqnetcyh:snapshot-1/MLdeB_rfTHSm1Pi94e247A] completed with state [SUCCESS]
[2021-02-05T11:31:36,787][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] --> unblocking [xtlqnetcyh] on node [node_t3]
[2021-02-05T11:31:36,833][INFO ][o.e.c.r.a.AllocationService] [node_t0] current.health="GREEN" message="Cluster health status changed from [YELLOW] to [GREEN] (reason: [shards started [[tsbcdvvlbr][0]]])." previous.health="YELLOW" reason="shards started [[tsbcdvvlbr][0]]"
[2021-02-05T11:31:36,843][INFO ][o.e.t.InternalTestCluster] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] Closing node [node_t3]
[2021-02-05T11:31:36,843][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] stopping ...
[2021-02-05T11:31:36,844][INFO ][o.e.c.c.Coordinator      ] [node_t3] master node [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}] failed, restarting discovery
org.elasticsearch.transport.NodeDisconnectedException: [node_t0][127.0.0.1:37113][disconnected] disconnected
[2021-02-05T11:31:36,845][INFO ][o.e.c.r.a.AllocationService] [node_t0] current.health="YELLOW" message="Cluster health status changed from [RED] to [YELLOW] (reason: [{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true} reason: disconnected])." previous.health="RED" reason="{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true} reason: disconnected"
[2021-02-05T11:31:36,845][INFO ][o.e.c.s.MasterService    ] [node_t0] node-left[{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true} reason: disconnected], term: 1, version: 24, delta: removed {{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true}}
[2021-02-05T11:31:36,851][INFO ][o.e.c.s.ClusterApplierService] [node_t1] removed {{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true}}, term: 1, version: 24, reason: ApplyCommitRequest{term=1, version=24, sourceNode={node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}
[2021-02-05T11:31:36,851][INFO ][o.e.c.s.ClusterApplierService] [node_t2] removed {{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true}}, term: 1, version: 24, reason: ApplyCommitRequest{term=1, version=24, sourceNode={node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}}
[2021-02-05T11:31:36,851][INFO ][o.e.c.s.ClusterApplierService] [node_t0] removed {{node_t3}{9T-YE4bRRZaiStnhF3Nupg}{mMg2b7NlSsywnEDvLthBDQ}{127.0.0.1}{127.0.0.1:33617}{d}{xpack.installed=true}}, term: 1, version: 24, reason: Publication{term=1, version=24}
[2021-02-05T11:31:36,851][INFO ][o.e.c.r.DelayedAllocationService] [node_t0] scheduling reroute for delayed shards in [59.9s] (3 delayed shards)
[2021-02-05T11:31:36,852][INFO ][o.e.c.r.a.AllocationService] [node_t0] current.health="RED" message="Cluster health status changed from [YELLOW] to [RED] (reason: [snapshot shard size updated])." previous.health="YELLOW" reason="snapshot shard size updated"
[2021-02-05T11:31:36,857][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] stopped
[2021-02-05T11:31:36,857][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] closing ...
[2021-02-05T11:31:36,859][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] closed
[2021-02-05T11:31:36,873][INFO ][o.e.s.SnapshotsService   ] [node_t0] deleting snapshots [old-version-snapshot-*] from repository [xtlqnetcyh]
[2021-02-05T11:31:36,873][INFO ][o.e.a.a.c.r.c.TransportCleanupRepositoryAction] [node_t0] Running cleanup operations on repository [xtlqnetcyh][0]
[2021-02-05T11:31:36,877][INFO ][o.e.a.a.c.r.c.TransportCleanupRepositoryAction] [node_t0] Done with repository cleanup on [xtlqnetcyh][0] with result [{"deleted_bytes":0,"deleted_blobs":0}]
[2021-02-05T11:31:36,883][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] [SearchableSnapshotsCanMatchOnCoordinatorIntegTests#testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped]: cleaning up after test
[2021-02-05T11:31:36,890][INFO ][o.e.c.m.MetadataDeleteIndexService] [node_t0] [tsbcdvvlbr/PstPc7i7Q9KpC858Yh440A] deleting index
[2021-02-05T11:31:36,890][INFO ][o.e.c.m.MetadataDeleteIndexService] [node_t0] [zddkaqnczh/jaUg9B9dSEqMK-Gh3y2GuQ] deleting index
[2021-02-05T11:31:36,907][INFO ][o.e.r.RepositoriesService] [node_t0] deleted repositories [[xtlqnetcyh]]
[2021-02-05T11:31:36,907][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] stopping ...
[2021-02-05T11:31:36,908][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] stopped
[2021-02-05T11:31:36,908][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] closing ...
[2021-02-05T11:31:36,908][INFO ][o.e.c.c.Coordinator      ] [node_t2] master node [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}] failed, restarting discovery
org.elasticsearch.transport.NodeDisconnectedException: [node_t0][127.0.0.1:37113][disconnected] disconnected
[2021-02-05T11:31:36,908][INFO ][o.e.c.c.Coordinator      ] [node_t1] master node [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}] failed, restarting discovery
org.elasticsearch.transport.NodeDisconnectedException: [node_t0][127.0.0.1:37113][disconnected] disconnected
[2021-02-05T11:31:36,908][INFO ][o.e.c.s.ClusterApplierService] [node_t1] master node changed {previous [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}], current []}, term: 1, version: 30, reason: becoming candidate: onLeaderFailure
[2021-02-05T11:31:36,908][INFO ][o.e.c.s.ClusterApplierService] [node_t2] master node changed {previous [{node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true}], current []}, term: 1, version: 30, reason: becoming candidate: onLeaderFailure
[2021-02-05T11:31:36,909][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] closed
[2021-02-05T11:31:36,909][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] stopping ...
[2021-02-05T11:31:36,909][WARN ][o.e.c.NodeConnectionsService] [node_t1] failed to connect to {node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true} (tried [1] times)
org.elasticsearch.transport.ConnectTransportException: [node_t0][127.0.0.1:37113] connect_exception
	at org.elasticsearch.transport.TcpTransport$ChannelsConnectedListener.onFailure(TcpTransport.java:953) ~[classes/:?]
	at org.elasticsearch.action.ActionListener.lambda$toBiConsumer$2(ActionListener.java:202) ~[classes/:?]
	at org.elasticsearch.common.concurrent.CompletableContext.lambda$addListener$0(CompletableContext.java:31) ~[classes/:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506) ~[?:?]
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2152) ~[?:?]
	at org.elasticsearch.common.concurrent.CompletableContext.completeExceptionally(CompletableContext.java:46) ~[classes/:?]
	at org.elasticsearch.nio.SocketChannelContext.connect(SocketChannelContext.java:131) ~[classes/:?]
	at org.elasticsearch.nio.EventHandler.handleConnect(EventHandler.java:106) ~[classes/:?]
	at org.elasticsearch.transport.nio.TestEventHandler.handleConnect(TestEventHandler.java:112) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.attemptConnect(NioSelector.java:417) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.registerChannel(NioSelector.java:440) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.setUpNewChannels(NioSelector.java:429) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.preSelect(NioSelector.java:250) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.singleLoop(NioSelector.java:144) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.runLoop(NioSelector.java:120) ~[classes/:?]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:589) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:839) ~[?:?]
	at org.elasticsearch.nio.SocketChannelContext.connect(SocketChannelContext.java:128) ~[classes/:?]
	... 9 more
[2021-02-05T11:31:36,910][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] stopped
[2021-02-05T11:31:36,910][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] closing ...
[2021-02-05T11:31:36,909][WARN ][o.e.c.NodeConnectionsService] [node_t2] failed to connect to {node_t0}{YY0YoUJzQvap0Pfc3efM1A}{EeqtWE1cQa2tgJllYRhzgg}{127.0.0.1}{127.0.0.1:37113}{m}{xpack.installed=true} (tried [1] times)
org.elasticsearch.transport.ConnectTransportException: [node_t0][127.0.0.1:37113] connect_exception
	at org.elasticsearch.transport.TcpTransport$ChannelsConnectedListener.onFailure(TcpTransport.java:953) ~[classes/:?]
	at org.elasticsearch.action.ActionListener.lambda$toBiConsumer$2(ActionListener.java:202) ~[classes/:?]
	at org.elasticsearch.common.concurrent.CompletableContext.lambda$addListener$0(CompletableContext.java:31) ~[classes/:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506) ~[?:?]
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2152) ~[?:?]
	at org.elasticsearch.common.concurrent.CompletableContext.completeExceptionally(CompletableContext.java:46) ~[classes/:?]
	at org.elasticsearch.nio.SocketChannelContext.connect(SocketChannelContext.java:131) ~[classes/:?]
	at org.elasticsearch.nio.EventHandler.handleConnect(EventHandler.java:106) ~[classes/:?]
	at org.elasticsearch.transport.nio.TestEventHandler.handleConnect(TestEventHandler.java:112) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.attemptConnect(NioSelector.java:417) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.registerChannel(NioSelector.java:440) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.setUpNewChannels(NioSelector.java:429) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.preSelect(NioSelector.java:250) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.singleLoop(NioSelector.java:144) ~[classes/:?]
	at org.elasticsearch.nio.NioSelector.runLoop(NioSelector.java:120) ~[classes/:?]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:589) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:839) ~[?:?]
	at org.elasticsearch.nio.SocketChannelContext.connect(SocketChannelContext.java:128) ~[classes/:?]
	... 9 more
[2021-02-05T11:31:36,910][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] closed
[2021-02-05T11:31:36,910][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] stopping ...
[2021-02-05T11:31:36,911][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] stopped
[2021-02-05T11:31:36,911][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] closing ...
[2021-02-05T11:31:36,912][INFO ][o.e.n.Node               ] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] closed
[2021-02-05T11:31:36,912][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] [SearchableSnapshotsCanMatchOnCoordinatorIntegTests#testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped]: cleaned up after test
[2021-02-05T11:31:36,912][INFO ][o.e.x.s.SearchableSnapshotsCanMatchOnCoordinatorIntegTests] [testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped] after test
REPRODUCE WITH: ./gradlew 'null' --tests "org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests.testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped" -Dtests.seed=5800A43D869703D3 -Dtests.locale=ms-MY -Dtests.timezone=America/Cuiaba -Druntime.java=14

java.lang.AssertionError: 
Expected: <6>
     but: was <3>
Expected :<6>
Actual   :<3>
<Click to see difference>


	at __randomizedtesting.SeedInfo.seed([5800A43D869703D3:372CAAED9F8FDB4A]:0)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests.testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped(SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java:327)
	at jdk.internal.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
	at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
	at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
	at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
	at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:824)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:475)
	at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
	at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
	at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
	at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
	at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
	at java.base/java.lang.Thread.run(Thread.java:832)
```

reliably after a few tens of iterations.

--- a/server/src/main/java/org/elasticsearch/indices/TimestampFieldMapperService.java
+++ b/server/src/main/java/org/elasticsearch/indices/TimestampFieldMapperService.java
@@ -112,6 +112,7 @@ public class TimestampFieldMapperService extends AbstractLifecycleComponent impl
                         protected void doRun() throws Exception {
                             try (MapperService mapperService = indicesService.createIndexMapperService(indexMetadata)) {
                                 mapperService.merge(indexMetadata, MapperService.MergeReason.MAPPING_RECOVERY);
+                                logger.trace("computed timestamp field mapping for {}", index);
                                 future.onResponse(fromMapperService(mapperService));
                             }
                         }

--- a/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java
+++ b/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java
@@ -14,8 +14,6 @@ import org.elasticsearch.action.search.SearchRequest;
 import org.elasticsearch.action.search.SearchResponse;
 import org.elasticsearch.cluster.metadata.DataStream;
 import org.elasticsearch.cluster.metadata.IndexMetadata;
-import org.elasticsearch.cluster.routing.IndexRoutingTable;
-import org.elasticsearch.cluster.service.ClusterService;
 import org.elasticsearch.common.Strings;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.common.unit.ByteSizeUnit;
@@ -321,16 +319,20 @@ public class SearchableSnapshotsCanMatchOnCoordinatorIntegTests extends BaseSear
         internalCluster().stopNode(dataNodeHoldingSearchableSnapshot);
         waitUntilAllShardsAreUnassigned(updatedIndexMetadata.getIndex());
 
-        SearchResponse newSearchResponse = client().search(request).actionGet();
+        // busy assert since computing the time stamp field from the cluster state happens off of the CS applier thread and thus can be
+        // slightly delayed
+        assertBusy(() -> {
+            SearchResponse newSearchResponse = client().search(request).actionGet();
 
-        // All the regular index searches succeeded
-        assertThat(newSearchResponse.getSuccessfulShards(), equalTo(totalShards));
-        assertThat(newSearchResponse.getFailedShards(), equalTo(0));
-        // We have to query at least one node to construct a valid response, and we pick
-        // a shard that's available in order to construct the search response
-        assertThat(newSearchResponse.getSkippedShards(), equalTo(totalShards - 1));
-        assertThat(newSearchResponse.getTotalShards(), equalTo(totalShards));
-        assertThat(newSearchResponse.getHits().getTotalHits().value, equalTo(0L));
+            // All the regular index searches succeeded
+            assertThat(newSearchResponse.getSuccessfulShards(), equalTo(totalShards));
+            assertThat(newSearchResponse.getFailedShards(), equalTo(0));
+            // We have to query at least one node to construct a valid response, and we pick
+            // a shard that's available in order to construct the search response
+            assertThat(newSearchResponse.getSkippedShards(), equalTo(totalShards - 1));
+            assertThat(newSearchResponse.getTotalShards(), equalTo(totalShards));
+            assertThat(newSearchResponse.getHits().getTotalHits().value, equalTo(0L));
+        });
     }
 
     public void testSearchableSnapshotShardsThatHaveMatchingDataAreNotSkippedOnTheCoordinatingNode() throws Exception {
@@ -502,10 +504,6 @@ public class SearchableSnapshotsCanMatchOnCoordinatorIntegTests extends BaseSear
     }
 
     private void waitUntilAllShardsAreUnassigned(Index index) throws Exception {
-        assertBusy(() -> {
-            ClusterService clusterService = internalCluster().getCurrentMasterNodeInstance(ClusterService.class);
-            IndexRoutingTable indexRoutingTable = clusterService.state().getRoutingTable().index(index);
-            assertThat(indexRoutingTable.allPrimaryShardsUnassigned(), equalTo(true));
-        });
+        awaitClusterState(state -> state.getRoutingTable().index(index).allPrimaryShardsUnassigned());
     }
 }

